english.txt

```diff
@@ -235,7 +235,7 @@ revivelink.com#$#abort-on-property-read btoa; abort-current-inline-script Object
 youdbox.com#$#abort-on-property-read encodeURIComponent
 vidshare.tv,vidshar.tv,series.movie,gowatchseries.movie,gowatchseries.tv#$#abort-current-inline-script Object.defineProperty XMLHttpRequest
 nosteam.ro#$#abort-on-property-read window.open
-manganelo.tv#$#abort-on-property-read document.createElement
+soap2day.rs,manganelo.tv#$#abort-on-property-read document.createElement
```

Comment:
@felippesantos46 please review this filter, for me it seems to break the video player, nothing can be played.

Reply:
yes, it works much better. 
Please resolve the PR conflicts and I'll merge them after.
thanks.